regEx2/challenges-11.test.js

```diff
@@ -0,0 +1,175 @@
+'use strict';
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 1
+//
+// Write a function named validatePin that uses a regular expression pattern to validate a PIN.
+//
+// If the PIN is four numerical digits long, return true. Otherwise, return false.
+// ------------------------------------------------------------------------------------------------
+
+const validatePin = (pin) => {
+  // Solution code here...
+  return /^[0-9]{4}$/.test(pin);
+
+  // if(/^[0-9]{4}$/.test(pin)) {
+  //     return true;
+  // } else {
+  //     return false;
+  // }
+};
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 2
+//
+// Write a function named findTagNames that iterates over an array of HTML strings
+// and uses a regular expression pattern to return the closing tags.
+//
+// For example, findTagNames(['<h1>Hello, world!</h1>', '<p>Welcome to my site</p>'])
+// returns ['/h1', '/p'], and findTagNames(['<div><h1>Hello, world!</h1></div>', '<p>Welcome to my site</p>'])
+// returns ['/h1', '/div', '/p'].
+// ------------------------------------------------------------------------------------------------
+
+const findTagNames = elements => {
+  // Solution code here...
+  let regex = /\/\w{2, 4}/g;
+  let answer = [];
+  elements.forEach( ele => {
+    answer.push(ele.match(regex));
```

Comment:
This is close! Each of these `match` calls will return an array, so you're currently returning an array of arrays. If you used `.concat` instead, you'd be golden!